### PR TITLE
Update the Related Product Rules page and add columns description

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request:
- updates the Related Product Rules page;
- adds columns description;

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/product-related-rules.html

### CODE REVIEW

File: src/marketing/product-related-rules.md [hunk 1]
```diff
@@ -5,5 +5,19 @@ title: Related Product Rules
...
 ![Related product rules]({% link images/images-ee/related-products-rules.png %}){: .zoom}
 _Related Products Rules_
+
+### Column descriptions
+
+|Column|Description|
+|--- |--- |
+|ID|A unique numeric identifier that is assigned to each related product rule|
+|Rule|The name of the related product rule|
+|Start|Use the dynamic calendar fields (To: and From:) to filter the list based on the start date for the rule as defined when the rule was created|
```
Comment (leslietilling) [inline src/marketing/product-related-rules.md:19]:
```suggestion
|Start|Use the dynamic calendar fields (_To:_ and _From:_) to filter the list based on the start date for the rule as defined when the rule was created.|
```

File: src/marketing/product-related-rules.md [hunk 2]
```diff
@@ -5,5 +5,19 @@ title: Related Product Rules
...
+|End|Use the dynamic calendar fields (To: and From:) to filter the list based on the end date for the rule as defined when the rule was created|
```
Comment (leslietilling) [inline src/marketing/product-related-rules.md:20]:
```suggestion
|End|Use the dynamic calendar fields (_To:_ and _From:_) to filter the list based on the end date for the rule as defined when the rule was created.|
```

File: src/marketing/product-related-rules.md [hunk 3]
```diff
@@ -5,5 +5,19 @@ title: Related Product Rules
...
+|Priority|Enter text in this field to filter the list based on the priority defined for a rule|
```
Comment (leslietilling) [inline src/marketing/product-related-rules.md:21]:
```suggestion
|Priority|Enter text in this field to filter the list based on the priority defined for a rule.|
```

File: src/marketing/product-related-rules.md [hunk 4]
```diff
@@ -5,5 +5,19 @@ title: Related Product Rules
...
+|Applies To|This option filters the list of rules that apply to: `Related Products`, `Up-sells`, `Cross-sells`|
```
Comment (leslietilling) [inline src/marketing/product-related-rules.md:22]:
```suggestion
|Applies To|This option filters the list of rules that apply to `Related Products`, `Up-sells`, and `Cross-sells`.|
```

File: src/marketing/product-related-rules.md [hunk 5]
```diff
@@ -5,5 +5,19 @@ title: Related Product Rules
...
+|Status|Use this option to filter the list based on rule status, `Active` or `Inactive`|
```
Comment (leslietilling) [inline src/marketing/product-related-rules.md:23]:
```suggestion
|Status|Use this option to filter the list based on rule status, `Active` or `Inactive`.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Just some minor edits. Some descriptions are complete sentences, so would need a period.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**